django_app/redbox_app/templates/homepage.html

```diff
@@ -8,8 +8,13 @@
     <div class="govuk-width-container">
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
+                {% if repo_owner | lower == "uktrade" %}
+                <h1 class="govuk-heading-xl govuk-!-margin-bottom-7">Use Artificial Intelligence (AI) to interact with Official level documents from your own document set</h1>
```

Comment:
This is exactly what the `MAX_SECURITY_CLASSIFICATION` setting is for toggling

Reply:
I've switched this to use security setting but checking if we still want specific differences in the wording. Will update once I hear back